lib/active_fedora/indexing.rb

```diff
@@ -87,15 +87,17 @@ def index_config
         # @param [Boolean] softCommit - Do we perform a softCommit when we add the to_solr objects to SolrService. Default true.
         # @param [Boolean] progress_bar - If true output progress bar information. Default false.
         # @param [Boolean] final_commit - If true perform a hard commit to the Solr service at the completion of the batch of updates. Default false.
-        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false)
+        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I don't think you can scope this to a method.  What I see in the docs is that you want to reinable it at the end of the method: http://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code